src/Sarif.Driver/Sdk/SdkResources.resx

```diff
@@ -120,48 +120,32 @@
   <data name="MSG1001_AnalyzingTarget_Description" xml:space="preserve">
     <value>An analysis target was analyzed.</value>
   </data>
-  <data name="ERR0998_ExceptionInAnalyze" xml:space="preserve">
-    <value>An exception was raised analyzing '{0}' for check '{1}' (which has been disabled for the remainder of the analysis). The exception may have resulted from a problem related to parsing image metadata and not specific to the rule, however. Exception information:
-{2}</value>
+  <data name="Notification_ExceptionInAnalyze" xml:space="preserve">
+    <value>An exception was raised analyzing '{0}' for check '{1}' (which has been disabled for the remainder of the analysis). The exception may have resulted from a problem related to parsing image metadata and not specific to the rule, however.</value>
```

Comment:
this whole parenthetical statement could merely consist of (which has been disabled).